src/current/v23.1/logging-overview.md

```diff
@@ -40,6 +40,7 @@ This allows you to group channels that log related information (e.g., operationa
 | [`SQL_EXEC`]({% link {{ page.version.version }}/logging.md %}#sql_exec)                 | SQL statement executions (when enabled via the `sql.trace.log_statement_execute` [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %})) and uncaught Go panic errors during SQL statement execution.                                                                                                                                    |
 | [`SQL_PERF`]({% link {{ page.version.version }}/logging.md %}#sql_perf)                 | SQL executions that impact performance, such as slow queries (when enabled via the `sql.log.slow_query.latency_threshold` and/or `sql.log.slow_query.experimental_full_table_scans.enabled` [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %})).                                                                                    |
 | [`TELEMETRY`]({% link {{ page.version.version }}/logging.md %}#telemetry)               |  Telemetry events for internal usage.                                                                                                                                                                                                                                        |
+| [`KV_DISTRIBUTION`]({% link {{ page.version.version }}/logging.md %}#kv_distribution)   | Data distribution events, such as moving replicas between stores in a cluster, adding replicas to ranges, and removing replicas from ranges. |
```

Comment:
Suggest adding the following links to first use of each term:

- 'replicas': https://www.cockroachlabs.com/docs/v23.1/architecture/overview#architecture-replica 
- 'range': https://www.cockroachlabs.com/docs/v23.1/architecture/overview#range
- 'stores': https://www.cockroachlabs.com/docs/v23.1/cockroach-start#store

Reply:
updated